content/en/docs/tasks/traffic-management/ingress/gateway-api/index.md

```diff
@@ -247,7 +247,7 @@ This resource does not actually exist in the cluster and is only used to signal
 For example, to redirect calls to `example.com` to an in-cluster `Service` named `example`:
 
 {{< text yaml >}}
-apiVersion: gateway.networking.k8s.io/v1beta1
+apiVersion: gateway.networking.k8s.io/{{< k8s_gateway_api_type >}}
```

Comment:
do we need to use k8s_gateway_api_type? it changes so rarely it seems like a hassle to have to use this.

Also only some types are beta, so we cannot actually use it universally

Reply:
We could leave this out. The issue didn't mention it, and I simply added both.

Reply:
Removed api_type